public/registerSW.js

```diff
@@ -0,0 +1 @@
+"serviceWorker"in navigator&&window.addEventListener("load",(async()=>{await navigator.serviceWorker.register("/sw.js")}));
```

Comment:
Should we use `. /` ? I'm using the nginx proxy with path `a.b.c/qb/`

Reply:
I just created a PR to address that issue:

https://github.com/VueTorrent/VueTorrent/pull/1511